docs/source/install.rst

```diff
@@ -25,28 +38,34 @@ Source
 
 The following instructions assume you'll be using ucx-py on a CUDA enabled system and is in a `Conda environment <https://docs.conda.io/projects/conda/en/latest/>`_.
 
-.. note::
-    UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, ``librdmacm``,
-    and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.
-    Additionally, we provide an example conda environment which installs necessary dependencies for building and testing ucx/ucx-py
 
+conda-forge Dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~
 
+::
 
+    conda create -n ucx-foo -c conda-forge \
+        automake make cmake libtool pkg-config \
+        libhwloc \
+        python=3.7 setuptools cython cupy distributed numba>=0.46 \
+        pytest pytest-asyncio \
+        ipython
 
-1) Create Conda Env
+RAPIDS Dependencies
```

Comment:
What do you think of not including RAPIDs dependencies in the install instructions.  We could mention this is the quickstart or examples but RAPIDS isn't necessary for installing ucx-py

Reply:
Well [RMM is used in our tests]( https://github.com/rapidsai/ucx-py/blob/e15bb7ce858bc3eb33fc29c3250d2e10e3b1402d/tests/test_send_recv_two_workers.py#L12 ). How should we track our test dependencies?

Though this does raise a good point. There are things like `cmake` and `ipython` that are not strictly dependencies either. Should they be dropped?

Reply:
I would say we drop everything that isn't really a hard dependency. If tests require additional libraries like RMM, then we can either say that or skip such tests when libraries aren't available.

Reply:
How about we break out the test dependencies? This way users and packagers can still see what is needed to run the tests.  🙂

As to the tests, I'll save that for another PR. Seems out-of-scope for a doc update 😉

Reply:
> How about we break out the test dependencies? This way users and packagers can still see what is needed to run the tests. 🙂

Seems good to me.

> As to skipping the tests, I'll save that for another PR. Seems out-of-scope for a doc update 😉

Absolutely, I didn't mean to suggest doing it here and now, just as the general solution that would be nice.

Reply:
Ok, I've given a go at this. PTAL 🙂

Agreed. It would be a good thing to do. It seems some tests use `pytest.importorskip`, but it's a bit inconsistent. Raised issue ( https://github.com/rapidsai/ucx-py/issues/352 ) to track this.

Reply:
I like the move to test dependencies.  Thanks @jakirkham